src/styles/blocks/product-card.scss

```diff
@@ -66,6 +78,8 @@
     border: 1px solid $blue-link-color;
     border-radius: 5px;
 
+    transition: color 400ms;
```

Comment:
add transition for background-color too

Reply:
Yeap, I just forget about it. Thanks for review!